src/store/mutations.ts

```diff
@@ -423,6 +425,34 @@ export default {
       ...payload,
     };
   },
+  SET_NFT_ASSETS(state: RootState, payload: NFTAssets) {
+    state.nftAssets = payload
```

Comment:
Aren't the nfts different depending on the network and walletid at the very least? what about depending on account? If so, then they need to be split in the state under those categories. See for example the `history` state object